packages/react/src/icon/index.test.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+import { mount, shallow } from 'enzyme';
+import { CdsIcon } from './index';
+import { ClarityIcons, userIcon, timesIcon } from '@clr/core/icon';
+import '@clr/core/icon/register';
```

Comment:
Since the React `CdsIcon` imports the register this import I don't think is needed